File: CS/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;

namespace WindowsFormsApp1 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            var report = new XtraReport1();
            report.ShowPreview();
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI

Namespace WindowsFormsApp1
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim report = New XtraReport1()
			report.ShowPreview()
		End Sub
	End Class
End Namespace
